devfiles/index.json

```diff
@@ -0,0 +1,52 @@
+[
+    {
+        "displayName": "Appsody Java MicroProfile template",
+        "description": "Microprofile using AdoptOpenJDK and Maven",
+        "language": "java",
+        "projectType": "appsodyExtension",
+        "location": "https://github.com/appsody/stacks/releases/download/java-microprofile-v0.2.2/incubator.java-microprofile.templates.default.tar.gz",
+        "links": {
```

Comment:
The links point to [devfiles](https://github.com/redhat-developer/devfile) that Eclipse Che uses to create a workspace.  At the moment, since appsody does not run on Kube and Che support is not there yet, we don't need the devfiles for now.